### DISCUSSION TITLE
Lb4 end of life

### DISCUSSION THREAD

**Discussion by ivanvaccari** [loopbackio/loopback-next#10403] [Q&A] [answered]
Hi.

As stated by the LTS page, https://loopback.io/doc/en/contrib/Long-term-support.html, loopback 4 appear to be on EOL at April 2024 (minimum). April 2024 is just  20 days away from now.

Are there any updates about the topic?
What' the current roadmap for future support?

**Comment by achrinza** [comment, marked as answer]
Hi @ivanvaccari thanks for raising this.

That page is out out of date. Loopback 4 is still the latest version, and hence it's EOL will be synced to the latest Node.js LTS version (2026-04-30).

The correct EOL date can be found here:

https://github.com/loopbackio/loopback-next/blob/master/README.md#long-term-support

Though we should update the website as well.
